File: Java/프로그래머스/0/120833. 배열 자르기/배열 자르기.java
class Solution {
    public int[] solution(int[] numbers, int num1, int num2) {
        int result[]=new int[num2-num1+1];
        for(int i=0; i<result.length;i++){
            result[i]=numbers[num1+i];
        }
        int[] answer = result;
        return answer;
    }
}

File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> numbers, int num1, int num2) {
    vector<int> answer(num2-num1+1);
    for(int i=0; i<size(answer) ;i++){
            answer[i]=numbers[num1+i];
    }
    return answer;
}